app/javascript/packs/src/settings/components/team/components/RemoveUserModal.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import PropTypes, { bool, number, string, func } from "prop-types";
+import { Modal, Button, Alert, Glyphicon } from "react-bootstrap";
+import { FormattedMessage, FormattedHTMLMessage } from "react-intl";
+
+import axios from "../../../../../app/axios";
+
+import { REMOVE_USER_FROM_TEAM_PATH } from "../../../../../app/routes";
+
+class RemoveUserModal extends Component {
+  constructor(props) {
+    super(props);
+    this.onCloseModal = this.onCloseModal.bind(this);
+    this.removeUser = this.removeUser.bind(this);
+  }
+
+  onCloseModal() {
+    this.props.hideModal();
+  }
+
+  removeUser() {
+    const { team_id, team_user_id } = this.props.userToRemove;
+    axios({
+      method: "DELETE",
+      url: REMOVE_USER_FROM_TEAM_PATH,
+      withCredentials: true,
+      data: {
+        team: team_id,
+        user_team: team_user_id
+      }
+    })
+      .then(response => {
+        this.props.updateUsersCallback(response.data.team_users);
+        this.props.hideModal();
+      })
+      .catch(error => console.log(error));
```

Comment:
just a reminder that we need to build global component that is connected to redux which would display this kind of errors to the user.